Cargo.toml

```diff
@@ -19,16 +19,16 @@ build = "build.rs"
 [dependencies]
 base64                = "^0.13"
 basic-cookies         = { version = "^0.1", optional = true }
-bcder                 = "0.5.0"
-bytes                 = "^0.5"
+bcder                 = "0.6.0"
```

Comment:
Even though Cargo will treat "0.6.0" as "^0.6.0", perhaps it is good to insert the caret explicitly for consistency with the other version values used in Cargo.toml.

Reply:
pushed